app/css/components/search-input.css

```diff
@@ -0,0 +1,5 @@
+@layer base {
+    .focused-input {
+        @apply bg-white border-lightBlue shadow-inputSelected;
+    }
+}
```

Comment:
This class should have the same name as the css file.

If it's a component, it should be prefixed with `c-`. If it's a style (which I think it is), it should be in ui-kit